plugins/inputs/mongodb/mongodb_server.go

```diff
@@ -36,9 +36,19 @@ func (s *Server) gatherData(acc telegraf.Accumulator) error {
 		log.Println("Not gathering replica set status, member not in replica set")
 	}
 
+	jumbo_chunks := 0
+	jumbo_chunks, err = s.Session.DB("config").C("chunks").Find(bson.M{"jumbo": true}).Count()
```

Comment:
Actually, it never panics, i just tested it with a non mongos nor mongo config instance and it works where the config DB doesn't exist, it just returns 0.
It is the same as when in the mongo shell you do 
````
use nonexistingdb
db.nonexisingcollection.find()
````

So, i also deleted the log message